resources/install-letsencrypt-cert.sh

```diff
@@ -1,135 +0,0 @@
-#!/bin/bash
-
-set -e
-
-DEB_CONF_RESULT=`debconf-show jitsi-meet-web-config | grep jvb-hostname`
```

Comment:
We should let users re-run this after the installation in case of issues. It happened to me several times that I had to run this more than once for it to work.

Reply:
That is already handled here https://github.com/jitsi/jitsi-meet/pull/12154/files#diff-888ed77886d118bbf50dfeb517849d2a05cc856426da1946aefb226342bb8adaR238

Reply:
I think I had problems even if the install let's encrypt script succeeded theoretically, or perhaps I did not see any error when running the install-letsencrpt-cert.sh script
Also, maybe the users start with self-signed certificate and decide later on to add let's encrypt. Then it would be helpful to continue to have this script for them to run.